lib/utils/version-from-zip.js

```diff
@@ -69,7 +69,10 @@ module.exports = function versionFromZip(zipPath, currentVersion, force = false)
         })
         .catch((err) => {
             // CASE: you are on latest v1, but you want to manually migrate to v2 using a zip
-            if (err.message.match(/No valid versions found/) && semver.major(currentVersion) !== semver.major(pkg.version)) {
+            // CASE: you are on v2 prerelease, and want to upgrade to the next prerelease
+            if (err.message.match(/No valid versions found/) &&
+                (semver.major(currentVersion) !== semver.major(pkg.version) || semver.prerelease(pkg.version))
```

Comment:
Logic looks correct. Thanks :)
Could you pls add one unit test case?

Reply:
Yep, oops 🙈

Reply:
fixed!